Formula/l/llgo.rb

```diff
@@ -1,8 +1,8 @@
 class Llgo < Formula
   desc "Go compiler based on LLVM integrate with the C ecosystem and Python"
   homepage "https://github.com/goplus/llgo"
-  url "https://github.com/goplus/llgo/archive/refs/tags/v0.9.9.tar.gz"
-  sha256 "705fed97ef8b337863fd9bbb40653c22fd93ba689f879db06801d37e5d8fe809"
+  url "https://github.com/goplus/llgo/archive/refs/tags/v0.10.0-pre.1.tar.gz"
```

Comment:
Is the `pre` in tag for prerelease? Though marked as GitHub latest. First time upstream used this format.

Reply:
yes, unusual indeed, may be worth waiting a bit with merge or asking upstream about it.

Reply:
Asked upstream in
* https://github.com/goplus/llgo/issues/972

Reply:
It also isn't showing up as latest via Go (`go install -v github.com/goplus/llgo/cmd/llgo@latest` and docs at https://pkg.go.dev/github.com/goplus/llgo)

Reply:
Interesting https://pkg.go.dev/github.com/goplus/llgo@v0.10.0-pre.1?tab=versions